db/1.8/modules/ROOT/pages/database-connector-examples.adoc

```diff
@@ -566,6 +566,46 @@ Use DataWeave for all data transformations before the invocation to the callable
 	</db:stored-procedure>
 ----
 
+[[use_pooling_profile_stored_procedures]]
+== Use Pooling Profile on Stored Procedures
+Because using a pooling profile provides fewer connections than the connections the flow uses and makes the entire flow hang, a plausible solution is to increase the size of the pool. However, sometimes this is not possible, so another workaround is to use the VM Connector to narrow the life of the connection to be like the life of a VM message. This way every call to the *Stored procedure* operation is executed, its result consumed, and the connection associated with the operation is closed and returned to the pool.
+
+The following example shows how to use pooling profiles with the *Store procedure* operation and VM Connector. When the foreach element reaches the eleventh iteration the *Stored procedure* operation won't hang since the pool is able to provide a connection:
```

Comment:
```suggestion
The following example shows how to use pooling profiles with the *Stored procedure* operation and VM Connector. When the `foreach` element reaches the eleventh iteration the *Stored procedure* operation won't hang since the pool is able to provide a connection:
```